README.md

```diff
@@ -58,7 +58,7 @@ After you're finished please remove all the comments and instructions!
 
 > This project is application project about movie description user of application can like and leave comments
 
-**[Movie Api]** is a javascript project
+**[Movie Api]** is a javascript project that fetches TV Show Data from an API
 
```

Comment:
Hi @MosDevx 
Good job so far!

To Highlight! 👏 🟢
- Good commit messages,
- Great folder structure
- Clean code
- Good PR description
- Your project is running free of Linter errors

There is some issue that you still need to work on  but you are almost there!
![](https://user-images.githubusercontent.com/65852621/133680327-431cb21d-ad7c-483a-84eb-21b733980a04.png)
Required Changes ♻️
Kindly add the link of live demo on your READEME file